scripts/hail_batch/main.py

```diff
@@ -0,0 +1,28 @@
+"""Run HGDP_1KG_PCALoadingsGenerate.py using the analysis runner."""
+
+import os
+import hail as hl
+import hailtop.batch as hb
+from analysis_runner import dataproc
+
+OUTPUT = os.getenv('OUTPUT')
+assert OUTPUT
+
+hl.init(default_reference='GRCh38')
+
+service_backend = hb.ServiceBackend(
+    billing_project=os.getenv('HAIL_BILLING_PROJECT'), bucket=os.getenv('HAIL_BUCKET')
+)
+
+batch = hb.Batch(name='generate PCA loadings', backend=service_backend)
+
+dataproc.hail_dataproc_job(
+    batch,
+    f'HGDP_1KG_PCALoadingsGenerate.py --output={OUTPUT}',
+    max_age='10h',
```

Comment:
Will this be enough? Would be a shame to have it stop after 10h with a partially completed computation.

Reply:
yes, that's a good point. I can bump it up a few hours (say 15?) only I was a bit concerned about cost. Do you know what the allocated budget is to the ancestry project?